src/views/domain/_modalPush.php

```diff
@@ -63,7 +63,7 @@
 <?php endif; ?>
 
 <?php foreach ($models as $model) : ?>
-    <?php if ($model->isPushable()) : ?>
+    <?php if ($model->canPush()) : ?>
```

Comment:
I would rename method to `canBePushed()`, because `If domain can be pushed` but not `If domain can push (someone?)`